bijection-core/src/test/scala/com/twitter/bijection/TypeclassBijectionLaws.scala

```diff
@@ -23,23 +23,6 @@ class TypeclassBijectionLaws
       def apply[A, B](tc: Ordering[A], bij: Bijection[A, B]) = tc.on { bij.invert(_) }
     }
 
-  implicit val numericTypeclassBijection: TypeclassBijection[Numeric] =
-    new TypeclassBijection[Numeric] {
-      def apply[A, B](tc: Numeric[A], bij: Bijection[A, B]) =
-        new Numeric[B] {
-          def plus(x: B, y: B) = bij(tc.plus(bij.invert(x), bij.invert(y)))
-          def minus(x: B, y: B) = bij(tc.minus(bij.invert(x), bij.invert(y)))
-          def times(x: B, y: B) = bij(tc.times(bij.invert(x), bij.invert(y)))
-          def negate(x: B) = bij(tc.negate(bij.invert(x)))
-          def fromInt(x: Int) = bij(tc.fromInt(x))
-          def toInt(x: B) = tc.toInt(bij.invert(x))
-          def toLong(x: B) = tc.toLong(bij.invert(x))
-          def toFloat(x: B) = tc.toFloat(bij.invert(x))
-          def toDouble(x: B) = tc.toDouble(bij.invert(x))
-          def compare(x: B, y: B) = tc.compare(bij.invert(x), bij.invert(y))
-        }
-    }
-
```

Comment:
Extending numeric requires an extra member in 2.13. Probably not worth it to split this test up, as typeclass testing seems adequately tested in other tests. Does it need additional replacements?